### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by falguniislam**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,1610 @@
...
+button, input {
+    overflow: visible;
+}
+button, select {
+    text-transform: none;
+}
+@media (min-width: 992px).navbar-expand-lg .navbar-toggler {
+    display: none;
+}
+
```
Comment (samgaco) [inline css/style.css]: The media queries should be enclosing the css classes they contain within curly brackets.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,1610 @@
...
+}
+@media (min-width: 992px)
+.navbar-expand-lg .navbar-collapse {
+    display: -ms-flexbox!important;
+    display: flex!important;
+    -ms-flex-preferred-size: auto;
+    flex-basis: auto;
+}
+
```
Comment (samgaco) [inline css/style.css]: They should be enclosing the classes they contain with curly brackets.
Example:
`@media(min-width:992px){
.class{ color: red;}
*class2{color:green;}
}`

**Comment by offorma** [comment]
Hi @falguniislam,
Great Job, At this moment you have successfully completed this project. So I give my thumbs up. You can go ahead and merge with the Master. Don't forget to submit a project completion form. I wish you happy learning in your next project.

**Comment by falguniislam** [comment]
Ok thank you very much

**Pull request merged**